decide_panel/src/components/Backups.js

```diff
@@ -0,0 +1,97 @@
+import { Card } from "@nextui-org/react";
+import { useRef } from "react";
+import { Messages } from "primereact/messages";
+import Api from "../services/backend";
+import "../css/ConnectionTest.css";
+import { useEffect, useState } from "react";
+import { Dropdown } from 'primereact/dropdown';
+
+
+const Backups = () => {
+  const [time, setTime] = useState(Date.now());
+  const [state, setState] = useState({
+    data: null,
+  });
+  const messages = useRef(null);
+
+  useEffect(() => {
+    const interval = setInterval(() => setTime(Date.now()), 1000);
+    return () => {
+      clearInterval(interval);
+    };
+  }, []);
+
+  useEffect(() => {
+    Api.get_backups()
+      .then((res) => setState({ data: res['availables backups'] }))
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645733)
```suggestion
      .then((res) => setState({ data: res["availables backups"] }))
```